src/Questionnaire/Questionnaire.js

```diff
@@ -0,0 +1,57 @@
+import React, { useState, useEffect } from 'react'
+import { questionnaires } from '../data/questionnaires'
+
+export default function Questionnaire() {
+  const currentQuestionnaire = questionnaires[0]
+  const { questions } = currentQuestionnaire
```

Comment:
Is this double-assignment necessary? Or is there a reason this doesn't work?
```suggestion
    const { questions }  = questionnaires[0]
```

Reply:
It isn't necessary. At the beginning I thought I should store which questionnaire is being answered, to make it easier to switch between questionnaires. But at this point your suggestion ist better.